pubsub/cloud-client/publisher_test.py

```diff
@@ -91,39 +98,41 @@ def test_publish(topic, capsys):
     publisher.publish_messages(PROJECT, TOPIC)
 
     out, _ = capsys.readouterr()
-    assert 'Published' in out
+    assert "Published" in out
 
 
 def test_publish_with_custom_attributes(topic, capsys):
     publisher.publish_messages_with_custom_attributes(PROJECT, TOPIC)
 
     out, _ = capsys.readouterr()
-    assert 'Published' in out
+    assert "Published" in out
 
 
 def test_publish_with_batch_settings(topic, capsys):
     publisher.publish_messages_with_batch_settings(PROJECT, TOPIC)
 
     out, _ = capsys.readouterr()
-    assert 'Published' in out
+    assert "Published" in out
 
 
 def test_publish_with_retry_settings(topic, capsys):
     publisher.publish_messages_with_retry_settings(PROJECT, TOPIC)
 
     out, _ = capsys.readouterr()
-    assert 'Published' in out
+    assert "Published" in out
 
 
 def test_publish_with_error_handler(topic, capsys):
     publisher.publish_messages_with_error_handler(PROJECT, TOPIC)
 
     out, _ = capsys.readouterr()
-    assert 'Published' in out
+    assert "Published" in out
 
 
 def test_publish_with_futures(topic, capsys):
     publisher.publish_messages_with_futures(PROJECT, TOPIC)
 
     out, _ = capsys.readouterr()
-    assert 'Published' in out
+    assert "Published" in out
+
+    _to_delete()
```

Comment:
I had to call delete out in `publisher_test.py` because there's a `test_delete()` and conflicts with teardown after `yield`. I left this comment earlier for Tim as well: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/2555#discussion_r354996508

Reply:
Why are the `eventually_consistent` necessary? Can they be changed in some way to make sure the resources are available by the time the tests are run so we can delete them through the fixtures?

Reply:
Maybe we can address this in a different PR? `@eventually_consistent` is used throughout the repo.

Reply:
Sounds good to me